clients/python/tests/test_client.py

```diff
@@ -77,6 +78,69 @@ def test_register_existing_version(client: ModelRegistry):
         client.register_model(**params)
 
 
+@pytest.mark.e2e
+async def test_update_models(client: ModelRegistry):
+    name = "test_model"
+    version = "1.0.0"
+    rm = client.register_model(
+        name,
+        "s3",
+        model_format_name="test_format",
+        model_format_version="test_version",
+        version=version,
+    )
+    assert rm.id
+
+    mr_api = client._api
+    mv = await mr_api.get_model_version_by_params(rm.id, version)
+    assert mv
+    assert mv.id
+    ma = await mr_api.get_model_artifact_by_params(name, mv.id)
+    assert ma
+
+    new_description = "updated description"
+    rm.description = new_description
+    mv.description = new_description
+    ma.description = new_description
+    assert client.update(rm).description == new_description
+    assert client.update(mv).description == new_description
+    assert client.update(ma).description == new_description
+
+
+@pytest.mark.e2e
+async def test_update_preserves_model_info(client: ModelRegistry):
+    name = "test_model"
+    version = "1.0.0"
+    uri = "s3"
+    model_fmt_name = "test_format"
+    model_fmt_version = "test_version"
+    rm = client.register_model(
+        name,
+        uri,
+        model_format_name=model_fmt_name,
+        model_format_version=model_fmt_version,
+        version=version,
+    )
+    assert rm.id
+
+    mr_api = client._api
+    mv = await mr_api.get_model_version_by_params(rm.id, version)
+    assert mv
+    assert mv.id
+    ma = await mr_api.get_model_artifact_by_params(name, mv.id)
+    assert ma
+
+    new_description = "updated description"
+    ma = ModelArtifact(id=ma.id, uri=uri, description=new_description)
+
+    updated_ma = client.update(ma)
+    assert updated_ma.description == new_description
+    assert ma.uri == updated_ma.uri
+    assert ma.id == updated_ma.id
```

Comment:
```suggestion
    assert updated_ma.uri == uri
    assert updated_ma.id == ma.id
```

just nit for more simmetry helpful when we will re-read this test in the future. wdyt?